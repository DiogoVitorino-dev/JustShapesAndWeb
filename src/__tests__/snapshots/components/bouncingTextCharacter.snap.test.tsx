import { render } from "test-utils";

import Character from "@/components/shared/bouncingText/character";

describe("Bouncing Text Character - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<Character start={false} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
