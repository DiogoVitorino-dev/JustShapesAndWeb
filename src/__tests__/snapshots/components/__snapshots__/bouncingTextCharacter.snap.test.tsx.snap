// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bouncing Text Character - Snapshot component test should render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
